tools/recetox_aplcms/recetox_aplcms_remove_noise.xml

```diff
@@ -39,7 +39,7 @@
     </inputs>
 
     <outputs>
-        <data label="${tool.name} on ${on_string}" name="output_file" format="parquet" />
+        <data label="${($input_file.element_identifier).rsplit('.',1)[0]}.parquet" name="output_file" format="parquet" />
```

Comment:
Could you maybe create an issue or some text with the underlying problem that y'all want to solve with this?